src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -87,16 +87,22 @@ public XmlLoggingConfiguration([NotNull] string fileName)
         /// <param name="fileName">Configuration file to be read.</param>
         /// <param name="logFactory">The <see cref="LogFactory" /> to which to apply any applicable configuration values.</param>
         public XmlLoggingConfiguration([NotNull] string fileName, LogFactory logFactory)
-            : this(fileName, false, logFactory)
-        { }
+            : base(logFactory)
+        {
+            using (XmlReader reader = CreateFileReader(fileName))
+            {
+                Initialize(reader, fileName);
+            }
+        }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="XmlLoggingConfiguration" /> class.
         /// </summary>
         /// <param name="fileName">Configuration file to be read.</param>
         /// <param name="ignoreErrors">Ignore any errors during configuration.</param>
+        [Obsolete("Constructor with parameter ignoreErrors is no longer supported and has no effect. Marked obsolete in NLog 5.0")]
```

Comment:
Maybe "limited effect" is more precise? Should we recommend using `LogFactory.ThrowConfigExceptions` /  `LogFactory.ThrowExceptions` ?

Reply:
I think this PR is waiting on this. I really like to merge this, it's a nice improvement!

Reply:
Wondering if the obsolete statement should be moved to NLog 4.7 (without code change) and then removed in NLog 5.0 ?

Reply:
The first is fine for me, the removal in 5.0 not - is too short in time for me. 

Removal in 6.0 would be fine.